gremlin-core/src/main/java/org/apache/tinkerpop/gremlin/process/traversal/dsl/graph/__.java

```diff
@@ -1306,41 +1306,83 @@ public static <A> GraphTraversal<A, A> range(final long low, final long high) {
         return __.<A>start().range(low, high);
     }
 
+    /**
+     * @see GraphTraversal#range(long, long)
```

Comment:
Doc nit, these should be updated.
```suggestion
     * @see GraphTraversal#range(GValue, GValue)
```

Reply:
Good catch, thanks.